pkg/gpu/nvidia/metrics/metrics.go

```diff
@@ -53,6 +54,30 @@ func (t *mCollector) collectDutyCycle(uuid string, since time.Duration) (uint, e
 }
 
 var (
+	// DutyCycle reports the percent of time when the GPU was actively processing per Node.
+	DutyCycleNodeGpu = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "duty_cycle_gpu_node",
+			Help: "Percent of time when the GPU was actively processing",
+		},
+		[]string{"node_name", "make", "accelerator_id", "model"})
+
+	// MemoryTotal reports the total memory available on the GPU per Node.
+	MemoryTotalNodeGpu = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "memory_total_gpu_node",
+			Help: "Total memory available on the GPU in bytes",
+		},
+		[]string{"node_name", "make", "accelerator_id", "model"})
+
+	// MemoryUsed reports GPU memory allocated per Node.
+	MemoryUsedNodeGpu = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "memory_used_gpu_node",
+			Help: "Allocated GPU memory in bytes",
+		},
+		[]string{"node_name", "make", "accelerator_id", "model"})
+
 	// DutyCycle reports the percent of time when the GPU was actively processing.
```

Comment:
nit: could you also update these comments to indicate they are metrics for per container?

Reply:
Done